mappings/net/minecraft/state/property/Property.mapping

```diff
@@ -1,6 +1,8 @@
 CLASS bwn net/minecraft/state/property/Property
 	METHOD a getName ()Ljava/lang/String;
-	METHOD a getValueAsString (Ljava/lang/Comparable;)Ljava/lang/String;
-	METHOD b getValueClass ()Ljava/lang/Class;
+	METHOD a getName (Ljava/lang/Comparable;)Ljava/lang/String;
```

Comment:
That method is clearly for getting the name of a value, IIRC, and should not be confused with getName(). As such, I'm saying no to this rename

Reply:
`property.getName(value)` is perfectly clear as to its intent imo.

Reply:
`getSerializedName`? Referring to that `StringRepresentable` horror.

Reply:
Or `nameValue`?

Reply:
Imo this method should be the same name as the property name getter. Both return a name. So if we rename this method the other should be reconsidered too. I personally think getName is fine though, anything else appended is just redundant verbosity.